server/courses/el/eisagwgh-sth-dhmosiografia-dedomenwn/index.yaml

```diff
@@ -0,0 +1,26 @@
+title: Introduction to Data Journalism
+introduction_html: |-
+    <p>
```

Comment:
Could you please translate this?

Reply:
I have already notified the translator that the file hasn't yet been translated.